scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -151,6 +151,7 @@ public boolean doCheckAndSend(boolean withAttachment)
             return false;
         }
 
+        JobEmailStatus jobEmailStatus = new JobEmailStatus(jobInfo, false, false);
```

Comment:
I see two improvement options here:
1/ Either you init jobEmailStatus only when necessary, i.e. in branchement with send e-mail
2/ Or, you keep this jobEmailStatus init here, but you configure it with jobEmailStatus.setWithErrors(jobEmailStatus.hasTasksWithErrors()) immediately, (proceed similarly w/ hasBeenAborted())

Reply:
I prefer the setter, personnally as well

Reply:
ok done